bridges/snowbridge/parachain/pallets/ethereum-beacon-client/Cargo.toml

```diff
@@ -1,11 +1,12 @@
 [package]
 name = "snowbridge-ethereum-beacon-client"
```

Comment:
Normally we have the work `pallet` in pallet crates, that makes it clear what kind of crate it is.

Reply:
@claravanstaden Lets rename it to `snowbridge-ethereum-client-pallet`. The `beacon` word is merely an inner inner implementation detail which we can forego.

Reply:
usually we have `pallet-*` as prefix so why not `pallet-snowbridge-ethereum-client` ?

Reply:
How about `snowbridge-pallet-ethereum-client`? It follows the same pattern as `cumulus-pallet-parachain-system`.

Reply:
Yea i think this makes the most sense.